src/events/klasaReady.ts

```diff
@@ -39,4 +40,11 @@ export default class extends Event {
 		}
 	}
 
+	private async initTwitchRefreshSubscriptionsTask() {
+		const { tasks } = this.client.schedule;
+		if (!tasks.some(task => task.taskName === 'twitchRefreshSubscriptions')) {
+			await this.client.schedule.create('twitchRefreshSubscriptions', '@daily');
```

Comment:
Might want to make this more common since it doesn't really cost anything to run a DB check for the times. At the same time you will eliminate blind spots if a user expires half a day before it runs and then streams.

Reply:
I think it's fine to run daily, but re-new those that are expiring in less than 24h.

Reply:
Any suggestions? It shouldn't be too often. DB entries are valid for 8 days from creation and Twitch subs for 9 (can be made 10).